### PR TITLE
Create 2018-02-19-thoughts-to-words-an-introduction-to-aaron-simumba.md

### PR DESCRIPTION
**Pull request by asimumba**
An introduction to Aaron Simumba personal website

### CODE REVIEW

**Comment by yihui** [comment]
You omitted the filename extension `.md`. Thanks!

**Comment by asimumba** [comment]
Rectified now.

**Comment by yihui** [comment]
Actually you only edited the title of this PR (not the filename).

**Comment by asimumba** [comment]
Sorry about that. I hope now all should be well.

**Comment by yihui** [comment]
No worries!

@rbind/matrix Can anyone review this post? The preview is at https://deploy-preview-158--rbind-support.netlify.com/2018/02/19/introduction-to-aaron-simumba-website/ Thanks!

**Comment by yihui** [comment]
Please remind me next week if no one reviews your PR by then. I'll review it. Thanks!

**Comment by joongsup** [comment]
I need a crash course reviewing/merging PR to be helpful for the next time. If the post looks good go ahead and merge PR?

**Comment by asimumba** [comment]
@yihui sure, I will.

**Comment by yihui** [comment]
@joongsup https://help.github.com/articles/about-pull-request-reviews/

**Comment by joongsup** [comment]
@asimumba @yihui I reviewed the post, and it looked good (maybe add a period at the end of the last sentence "...you can find it here"?). I like your landing home page, esp the changing images! Also this may not be in the scope of PR review (and it could easily be just me), but I couldn't see the thumbnail images in your travel page (https://asimumba.rbind.io/travel/) if they are supposed to be there.

**Comment by yihui** [comment]
@joongsup You can click the line numbers to make specific comments. In the end, you need to make a decision among the three choices:

image

File: content/post/2018-02-19-thoughts-to-words-an-introduction-to-aaron-simumba-s-website.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+I love the post preview from the blog section. It gives a brief section to preview the content before delving fully into the post. From the original theme structure, this is presented on the front page if you use `/content/post/` structure. To avoid this, I changed it to `/content/blog/`. All my posts reside in the `/blog` subdirectory.
+
+![blog](https://user-images.githubusercontent.com/24398851/36397317-016df4c2-15cb-11e8-986f-b33a31689bc6.jpg)
+
+# End goal
+
+Going forward, I will use this website to document my R learning path. And occasionally anything I find remotely interesting for the future me for reference purposes. More like turning my wild thoughts to written engravings. Happy blogging...
+
+I cannot document all the small but significant changes I made. The best option would be to fork the site and have a look at the changes made. For the source to my website, you can find it [here](https://github.com/rbind/asimumba)
```
Comment (joongsup) [inline content/post/2018-02-19-thoughts-to-words-an-introduction-to-aaron-simumba-s-website.md]: Small thing, but maybe an ending period at the end of the sentence?

Comment (asimumba) [reply content/post/2018-02-19-thoughts-to-words-an-introduction-to-aaron-simumba-s-website.md]: @joongsup Thanks for pointing that out. I have rectified the missing period. On the thumbnails on images. I will have to tweak the theme to make it compatible with the aspect ratio of my images. Actually the images are there if you hover over the the blank space. Due to the theme restriction, images have to be set to a certain specified length to width ratio.

**Pull request merged**